config/env.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
```

Comment:
assuming all this `config/*` stuff is from CRA eject

Reply:
Yup! It's quite a lot and is probably way too complex for what we need, but it works. :woman_shrugging:

Reply:
Wow. This is more scaffolding than I thought there would be. How do you feel about this? Do you think it's too confusing having all this "extra" stuff in the repo?

Reply:
Wow. This is more scaffolding than I thought there would be. How do you feel about this? Do you think it's too confusing having all this "extra" stuff in the repo?

Reply:
I agree that it is way too much for our projects. Not sure if it's worth spending time to add a custom webpack now or do some thinking about our future tools. There are other options available that might be better?